devtools-ci-index.yaml

```diff
@@ -1214,9 +1227,11 @@
 
             export GIT_COMMIT=$UPSTREAM_GIT_COMMIT
 
+            echo "$KUBE_CONFIG" | base64 -d > kube_config
+
             # What's the current deployment revision?
             export DC_NAME=${{DEPLOYMENT_CONFIGS##* }}
-            DC_REVISION=$(oc get dc/$DC_NAME -n $PRJ_NAME |grep $DC_NAME | awk '{{ print $2 }}')
+            DC_REVISION=$(oc --config=kube_config get dc/$DC_NAME -n $PRJ_NAME |grep $DC_NAME | awk '{{ print $2 }}')
```

Comment:
This job-template has a particularity. The `kube_config` file is written twice, but I think that's not a problem, right @riuvshin?

This would be the resulting script:

Reply:
https://gist.github.com/jmelis/b7f3dc3ce044369fe30671b02160f345#file-devtools-f8a-master-deploy-e2e-test-scrpt-sh-L13
https://gist.github.com/jmelis/b7f3dc3ce044369fe30671b02160f345#file-devtools-f8a-master-deploy-e2e-test-scrpt-sh-L46